src/QueryBuilder.php

```diff
@@ -723,6 +723,7 @@ protected function getDuplicatesQuery()
 
         $waFirst = $this->query->getGrammar()->wrapTable($firstAlias);
         $waSecond = $this->query->getGrammar()->wrapTable($secondAlias);
+        $pk = $this->model->getKeyName();
```

Comment:
Why this line was added?

Reply:
Guess cause primary key can also be something else rather than just id, but this shouldn't be a part of a L6 upgrade PR, but doesn't hurt to have the functionality.

Reply:
But `$pk` variable is never used in this class.

Reply:
See https://github.com/lazychaser/laravel-nestedset/pull/399/commits/f648e15d955e722692e7b8696fb3a3a3b9313c2d

Reply:
It looks latest @mewebstudio
https://github.com/lazychaser/laravel-nestedset/blob/102630719e2dbffea7193b95d1a16c3f49b6d01b/src/QueryBuilder.php

And there is no $pk in master branch https://github.com/lazychaser/laravel-nestedset/blob/v4/src/QueryBuilder.php

Reply:
Agreed, it is weird, this update should just have the version bumb so we can get l6 compatibility today

Reply:
I created a PR with just the laravel 6 update stuff, https://github.com/lazychaser/laravel-nestedset/pull/400

Reply:
Hi.